webpack.config.js

```diff
@@ -104,16 +104,23 @@ if (process.env.NODE_ENV !== 'dev' && process.env.NODE_ENV !== 'auto') {
 
 const entry = {
   "bundle": ['babel-polyfill', './src/index.jsx'],
-  "workspaceBundle": ['babel-polyfill', './src/workspaceIndex.jsx']
+  "workspaceBundle": ['babel-polyfill', './src/workspaceIndex.jsx'],
+  "covid19Bundle": ['babel-polyfill', './src/covid19Index.jsx']
 };
 
 if (process.env.GEN3_BUNDLE === 'workspace') {
```

Comment:
how would we enable multiple bundles?

Reply:
if you don't specify `GEN3_BUNDLE` value then it will build all bundles, is that what you mean?

Reply:
i see, what i was thinking is that we could have `GEN3_BUNDLE=covid19,workspace`

but then i'm not sure how this `entry.bundle` stuff would work 😅

Reply:
oh cool. I don't think we have the need to deploy multiple bundles right now so right now if you spin up a portal pod you would have to choose one flavor only

Reply:
that's fair, then follow up question, why don't we do
```
if (process.env.GEN3_BUNDLE === 'workspace') {
  entry.bundle = ['babel-polyfill', './src/workspaceIndex.jsx'];
}
```
and default to only building `./src/index.jsx`, instead of deleting entries which looks a bit messy?

Reply:
I think for `Dockerfile` we don't have `GEN3_BUNDLE` set so it will build all bundles to make sure nothing stops us from building an image. So we can either `delete` from a full `entry` array, or adding into it for that case